ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_tensor.py

```diff
@@ -2322,6 +2381,40 @@ def test_paddle_tensor_greater_than(
     )
 
 
+# histogram
+@handle_frontend_method(
+    class_tree=CLASS_TREE,
+    init_tree="paddle.to_tensor",
+    method_name="histogram",
+    values=_histogram_helper(),
```

Comment:
`TypeError: _histogram_helper() missing 1 required positional argument: 'draw'`
could you fix that please?